pkg/clusterd/inventory/memory.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package inventory
+
+// #include <unistd.h>
+import "C"
+
+import (
+	"fmt"
+	"path"
+	"strconv"
+
+	etcd "github.com/coreos/etcd/client"
+	ctx "golang.org/x/net/context"
+)
+
+func getSystemMemory() uint64 {
+	return uint64(C.sysconf(C._SC_PHYS_PAGES) * C.sysconf(C._SC_PAGE_SIZE))
```

Comment:
this is the standard way to get memory size from go code?  Does this new cgo dependency affect our build in any way we need to be aware of?